### ISSUE TITLE
pos_weight argument in torch.nn.BCELoss

### ISSUE THREAD

**Issue by FedericoMuciaccia** [pytorch/pytorch#20424]
## 🚀 Feature requested
I'd like to have a `pos_weight` arguments in `torch.nn.BCELoss`

## Motivation
to have consistency with `torch.nn.BCEWithLogitsLoss` and to be able to address a multi-label task with a strong class imbalance

**Comment by peter850706** [comment]
Agree with @FedericoMuciaccia. Is there any updates for this?

**Comment by ciaochiaociao** [comment]
Second this
